queries.sql

```diff
@@ -8,3 +8,36 @@ SELECT  name, escape_attempts from animals WHERE   weight_kg > 10.5;
 SELECT * from animals WHERE   neutered = true;
 SELECT * from animals WHERE   name <> 'Gabumon';
 SELECT * from animals WHERE  weight_kg >= 10.4 and weight_kg  <= 17.3;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species is null;
+UPDATE animals SET species = 'pokemon' WHERE species is null;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
```

Comment:
Some query codes are missing 👇🏾 
- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
- Inside a transaction delete all records in the animals table, then roll back the transaction; After the rollback verify if all records in the animals table still exists.
Take a screenshot of the results of your actions.

- Inside a transaction:
  * `DELETE` all animals born after Jan 1st, 2022
  * Create a savepoint for the transaction
  * `UPDATE` all animals' weight to be their weight multiplied by -1.
  * Rollback to the savepoint
  * `UPDATE` all animals' weights that are negative to be their weight multiplied by -1.
  * Commit transaction

### Please add the above missing query codes to the codebase. Please put label to all your query codes like so 👇🏾 
Please Note that the last two queries above are different, hence the should be executed differently with proper labels.

```sql
-- UPDATE the animals table by setting the species column to unspecified, then roll back the change
BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone

-- Add the query question here as label
BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```

Reply:
Thank you once again for such progressive advise, adding labels is a good practice! 🥇  👍  @bobb-Rob